admin_manual/configuration_files/external_storage/smb.rst

```diff
@@ -90,3 +90,9 @@ Hidden files upload failure or not shown
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 If you have the configuration ``hide dot files = Yes``, you will not be able to upload a hidden file (dot file) nor will you be able to show hidden files on your filelist (even if the 'show hidden file' option is checked on the nextcloud settings.
 Make sure you have the following option in your configuration: ``hide dot files = No``
+
+Using ACL support
+^^^^^^^^^^^^^^^^^
+When enabling ACL support for external SMB/CIFS storage, Nextcloud can respect filesystem-level access controls. 
```

Comment:
This should clarify that it only impacts how the permissions are represented in the UI.